test/passing/doc_comments.mli

```diff
@@ -13,23 +13,25 @@ val escape : string -> string
 
 type title = string * int * string * string * string
 
-(** {1 Standard section names} *)
+(** {1:standard-section-names Standard section names} *)
```

Comment:
Is it necessary to add `:standard-section-names`? Seems very redundant.

Reply:
This is not optional in Odoc's ast. Work is needed to remove them.

Reply:
Ok I see. Are they generated if omitted, so that we could detect if they have the generated name and then not print them?

Reply:
Users may want to explicitly add the label. Reading the original string is possible but tricky. A good solution would be to modify Odoc.